File: Exercise_Make a Stopwatch.R
#Prompt: Write a program that prints out the time it takes to run the program.

#set start time
stime <- proc.time()

#code to do something
print('hello word')

#set end time
etime <- proc.time()

#time that elapsed
tlapsed <- etime - stime

#display
print(tlapsed)

File: Exercise_Make a Stopwatch.py
# -*- coding: utf-8 -*-
"""
Created on Thu May 13 14:32:07 2021

@author: Situa
"""
#Prompt: Write a program that prints out the time it takes to run the program.

#invoke time module
import time

#set start time
stime = time.time()

#set end time
etime = time.time()

#time in between start and end
tlapsed = etime - stime

#display
print(tlapsed, "seconds")



